### PR TITLE
Translate guide - migration - interpolations

### PR DESCRIPTION
**Pull request by Tintef**

### CODE REVIEW

File: src/v2/guide/migration.md [hunk 1]
```diff
@@ -1194,23 +1194,23 @@ When used together with `<transition>`, make sure to nest it inside:
...
 {% endraw %}
 
-## Interpolation
+## Interpolaciones
 
-### Interpolation within Attributes <sup>removed</sup>
+### Interpolaciones dentro de Atributos <sup>removido</sup>
 
-Interpolation within attributes is no longer valid. For example:
+La interpolación dentro de atributo ya no es válida. Por ejemplo:
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
La interpolación dentro de atributos ya no es válida. Por ejemplo:
```

File: src/v2/guide/migration.md [hunk 2]
```diff
@@ -1194,23 +1194,23 @@ When used together with `<transition>`, make sure to nest it inside:
...
 
 ``` html
 <button class="btn btn-{{ size }}"></button>
 ```
 
-Should either be updated to use an inline expression:
+Debería actualizarse a utilizar una expresión _inline_:
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]:
```suggestion
Debería actualizarse para utilizar una expresión _inline_:
```

File: src/v2/guide/migration.md [hunk 3]
```diff
@@ -1226,30 +1226,32 @@ computed: {
...
 
-One time bindings (`{% raw %}{{* foo }}{% endraw %}`) have been replaced by the new [`v-once` directive](../api/#v-once).
+Los enlaces _one-time_ (`{% raw %}{{* foo }}{% endraw %}`) han sido reemplazados por la nueva [directiva `v-once`](../api/#v-once).
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find one-time bindings.</p>
+  <h4>Ruta de actualización</h4>
+  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar enlaces <i>one-time</i>.</p>
```
Comment (sxcamacho) [inline src/v2/guide/migration.md:1254]:
```suggestion
  <p>Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para encontrar enlaces _one-time_.</p>
```

**Review by sxcamacho — CHANGES_REQUESTED** [review]

File: src/v2/guide/migration.md (hunk 3 above)
Comment (Tintef) [reply src/v2/guide/migration.md:1254]: @sxcamacho Actually, as this is inside a `{% raw %}` we need to use `<i>` in order to make that text appear in italics

**Comment by Tintef** [comment]
@sxcamacho Thanks for the review! Everything updated! 😄

File: src/v2/guide/migration.md (hunk 3 above)
Comment (sxcamacho) [reply src/v2/guide/migration.md:1254]: You're right, disregard the suggestion

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**